mvc.js

```diff
@@ -0,0 +1,134 @@
+class Model {
+    constructor() {
+        this.data = JSON.parse(localStorage.getItem("signUp")) || [];
+    }
+
+    setData(user) {
+        localStorage.setItem("signUp", JSON.stringify(user));
+    }
+
+    addData(email, password) {
+        const newUser = {
+            email,
```

Comment:
in Sign Up form you have 5 fields. I think you should push 4 fields to storage(first and last name, email  and password)Confirm is not necessary

Reply:
Resolved